package.json

```diff
@@ -50,6 +50,11 @@
         "command": "watermelon.blame",
         "title": "Get Commit History with Watermelon",
         "category": "watermelon"
+      },
+      {
+        "command": "watermelon.docs",
```

Comment:
Is there a handler for this?

Reply:
A quick test responded my question.
At least the user gets feedback ![image](https://user-images.githubusercontent.com/11527621/171689794-f9052ad9-3e6f-4b59-a6f3-291eb88c7003.png)